### DISCUSSION TITLE
Is it possible to continue to watch a video from the moment I paused it?

### DISCUSSION THREAD

**Discussion by oren** [FreeTubeApp/FreeTube#4257] [Q&A]
After closing a video without completing watching it I sometime want to continue and watch from the point in time where I paused it. I looked at the settings and couldn't find an option. I am surprised that it's not the default.

Any ideas?

**Comment by absidue** [comment]
In the privacy settings turn on watch history and watch progress saving.

It's disabled by default firstly because some people don't want it (me for example, those kinda of things provide no benefit to me and are just annoying) and secondly it would be quite odd for a privacy friendly app, to default to keeping an exact record of when you watched what by default. Those kind of things should always be opt-in, not opt-out.
